tests/unit/utils.spec.js

```diff
@@ -80,15 +76,28 @@ describe('Testing Utils', function() {
       });
     });
 
+    it('should return custom error when called with custom error messages', function() {
+      let customErrorResponse = {
+        status: 'Access Denied',
+        message: 'You are not authorized to access this resource'
+      };
+
+      utils.deny(res, 403, null, customErrorResponse);
+      let data = res._getData();
+      assert(data);
+      expect(data).to.be.an('object');
+      expect(res.statusCode).to.equal(403);
+      assert.deepEqual(data, customErrorResponse);
+    });
+
   });
 
   describe('Utils.whenGlobAndActionAllow', function() {
     context('When the Methods are a string', function() {
 
       it('should call next when method is string and *', function() {
         utils.whenGlobAndActionAllow(res, next, null, '*');
-
-        data = res._getData();
+        let data = res._getData();
```

Comment:
[**Nitpick**] @andela-thomas Should we declare `data` on the upper scope (line 9) to avoid repetition?